_maps/shuttles/emergency_zeta.dmm

```diff
@@ -178,8 +178,7 @@
 "E" = (
 /obj/structure/table/abductor,
 /obj/item/crowbar/abductor{
-	pixel_x = -13;
-	pixel_y = 0
```

Comment:
Just gonna leave this here as a remind that I can't determine the difference between a warning and an error.